pipeline/git/git.go

```diff
@@ -30,5 +34,7 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		PreviousTag: previous,
 		Diff:        log,
 	}
+	// removes usual `v` prefix
+	ctx.Version = strings.TrimPrefix(tag, "v")
```

Comment:
That's a little too opinionated for my taste. 
I think we should also allow users to use a custom version scheme. Maybe we can change it to not error and instead ignore other types of formats?